examples/eco/07-integrated-hydrogen/input/plant/orbit-config-osw_18MW.yaml

```diff
@@ -0,0 +1,231 @@
+    electrolyzer: 2020 # 2020 for singlitico2021, 2016 # for simple h2 cost model in hopp (see https://www.hydrogen.energy.gov/pdfs/19009_h2_production_cost_pem_electrolysis_2019.pdf) ## 2020 # based on IRENA report https://www.irena.org/-/media/Files/IRENA/Agency/Publication/2020/Dec/IRENA_Green_hydrogen_cost_2020.pdf
+    h2_transport_compressor: 2016 # listed in code header 
+    h2_storage:
+      pressure_vessel: 2022 # based on readme for Compressed_gas_function
+      pipe: 2019 # Papadias 2021
+      salt_cavern: 2019 # Papadias 2021
+      turbine: 2003 # assumed based on Kottenstette 2004
+      none: 2022 # arbitrary
+    h2_pipe_array: 2018 # ANL costs
+    h2_transport_pipeline: 2018 # same model for costs as the h2_pipe_array
+
+electrolyzer:
+  rating: 180 # MW # 0.9*Plant rating appears near-optimal for 400 MW wind plant with 3 days of underground pipe storage # MW
+  electrolyzer_capex: 700 # $/kW conservative 2025 centralized. high 700, low 300 # based on https://www.irena.org/-/media/Files/IRENA/Agency/Publication/2020/Dec/IRENA_Green_hydrogen_cost_2020.pdf
+  time_between_replacement: 62320 # 62320 based on PEM Electrolysis H2A Production Case Study Documentation estimate of 7 years, 40000 based on current est (see unpublished report), 80000 hours based on OSW atb_year = 2025
+  replacement_cost_percent: 0.15 # percent of capex - H2A default case
+  model: "singlitico2021" # "basic" is a basic cost model based on H2a and HFTO program record for PEM electrolysis. "singlitico2021" uses cost estimates from that paper
+# h2_transport_compressor:
+#   output_pressure: 100 # bar (1 bar = 100 kPa)
+#   flow_rate: 89 # kg/hr
+#   energy_rating: 802 # kWe (aka 1 kWh)
+#   mean_days_between_failures: 200 # days
+#   useful_life: 15 # was default in compressor script
+#   # annual_h2_throughput: 18750 # [kg/yr] -> kg of H2 per year 
+h2_transport_compressor:
+  outlet_pressure: 68 # bar based on HDSAM
+h2_storage_compressor:
+  output_pressure: 100 # bar (1 bar = 100 kPa)
+  flow_rate: 89 # kg/hr
+  energy_rating: 802 # kWe (aka 1 kWh)
+  mean_days_between_failures: 200 # days
+  # annual_h2_throughput: 18750 # [kg/yr] -> kg of H2 per year 
+h2_transport_pipe:
+  outlet_pressure: 10 # bar - from example in code from Jamie #TODO check this value
+h2_storage:
+  # capacity_kg: 18750 # kg
+  capacity_from_max_on_turbine_storage: True # if True, then days of storage is ignored and storage capacity is based on how much h2 storage fits on the turbines in the plant using Kottenstete 2003.
+  type: "none" # can be one of ["none", "pipe", "turbine", "pressure_vessel", "salt_cavern"] ### not yet implemented:  ["line_rock_cavern"] #TODO fix salt cavern, just a number right now
+  days: 3 # [days] how many days worth of production we should be able to store (this is ignored if `capacity_from_max_on_turbine_storage` is set to True)
+pv:
+  flag: False
+  system_capacity_kw: 180000 # TODO stop this number from printing when HOPP is run
```

Comment:
Is this still happening? Can we remove this comment?

Reply:
I removed all old print statements and output seems clean to me now.

Reply:
Comment has been removed